scripts/install

```diff
@@ -249,7 +240,7 @@ npm install ep_automatic_logut@${ep_automatic_logut_version} >> $install_log 2>&
 # Framapad - Adds comments on sidebar and link it to the text.
 npm install ep_comments_page@${ep_comments_page_version} >> $install_log 2>&1
 # Framapad - Displays paragraphs, sentences, words and characters counts.
-#npm install ep_countable@${ep_countable_version} >> $install_log 2>&1
+npm install ep_countable@${ep_countable_version} >> $install_log 2>&1
```

Comment:
You should look at https://yunohost.org/fr/packaging_apps_helpers?q=%2Fpackaging_apps_helpers#nodejs, you can use `ynh_npm` instead of `npm`